spec/membrane_element_lame/lame_spec.exs

```diff
@@ -107,7 +107,7 @@ defmodule Membrane.Element.Lame.EncoderSpec do
 
         it "should return an ok result" do
           {result, _new_state} = handle_process()
-          expect(result) |> to(eq :ok)
+          expect(result) |> to(match_pattern {:ok, _})
```

Comment:
That's too permissive, write this test properly. Here you would probably want to make sure no buffer was returned